packages/protocol/runTests.js

```diff
@@ -16,6 +16,12 @@ const migrationOverrides = {
   downtimeSlasher: {
     slashableDowntime: 60, // epoch length is 100 for unit tests
   },
+  governanceApproverMultiSig: {
+    signatories: [network.from],
```

Comment:
nit: Consider including `numRequiredConfirmations` here as well, so that if the `migrationsConfig` changed to e.g. 2 the tests would still work